cabal.project.stage1

```diff
@@ -67,9 +66,14 @@ package *
 
 package ghc
   flags: +bootstrap
-
-package ghci
-  flags: +bootstrap
+         +x86-ncg +aarch64-ncg
+         -ppc-ncg -riscv64-ncg -loongarch64-ncg
+         -js-backend -wasm-backend
+         +llvm-backend
+         -interpreter -dynamic-linker
```

Comment:
I think we might want to disable `llvm-backend` by default. And only enable `+x86-ncg` with a conditional`if arch(x86)`, same for `aarch64`?